app/domain/authentication/authn_gcp/validate_resource_restrictions_match_jwt.rb

```diff
@@ -38,14 +38,26 @@ def resource_type_without_prefix(resource_type)
       def resource_value_from_token(resource_type)
         case resource_type
         when PROJECT_ID_RESTRICTION_NAME
-          @decoded_token.project_id
+          resource_value = @decoded_token.project_id
         when INSTANCE_NAME_RESTRICTION_NAME
-          @decoded_token.instance_name
+          resource_value = @decoded_token.instance_name
         when SERVICE_ACCOUNT_ID_RESTRICTION_NAME
-          @decoded_token.service_account_id
+          resource_value = @decoded_token.service_account_id
         when SERVICE_ACCOUNT_EMAIL_RESTRICTION_NAME
-          @decoded_token.service_account_email
+          resource_value = @decoded_token.service_account_email
```

Comment:
No else in case?

Reply:
No need, we already have this conversation